### ISSUE TITLE
Buttons at the top of the main view display d,g,e,f,b instead of symbols

### ISSUE THREAD

**Issue by joshbtn** [johanlunds/atom-ruby-debugger#9]
Buttons at the top of the main view display d,g,e,f,b instead of symbols.

Looks like there are built in symbols that will work from the atom styleguide.  (Inside an atom workspase press `ctrl + shift + g` on windows or search for 'styleguide')

**Comment by johanlunds** [comment]
They are working for me, though the margins are not correct.

<img width="1251" alt="_users_johan_lunds_documents_kod_atom-ruby-debugger_-_atom_and_1__johan_lunds_johan-l-fishbrain-macbook-pro____documents_kod_atom-ruby-debugger__zsh_" src="https://cloud.githubusercontent.com/assets/22144/13610927/4b50b482-e560-11e5-8756-61afc566bdc5.png">

**Comment by johanlunds** [comment]
Because I like the Chrome debugger I had taken the debugger-symbols from that project and packaged them up as a Webfont in https://github.com/johanlunds/atom-ruby-debugger/tree/master/resources.

I generated the webfont using http://fontastic.me/ using the resources here: https://github.com/johanlunds/atom-ruby-debugger/tree/master/docs/icons

**Comment by johanlunds** [comment]
Are you using Windows? I am using Mac.

**Comment by joshbtn** [comment]
I am using windows, and I can see an error when it tries to load in the fonts.  I'll dive in a little further and test on my mac later.

**Comment by joshbtn** [comment]
I see this on my mac as well.  It may be since I just checked the repo out into my .atom/packages folder...

![screen shot 2016-03-08 at 6 45 27 pm](https://cloud.githubusercontent.com/assets/562773/13620480/12062786-e55e-11e5-8c14-b1bea52e240f.png)
![screen shot 2016-03-08 at 6 46 04 pm](https://cloud.githubusercontent.com/assets/562773/13620481/1548c3f4-e55e-11e5-9839-4a920d6fef5e.png)

**Comment by johanlunds** [comment]
Try this; this is what I did:

```
cd ~/Code # or wherever
git clone git@github.com:johanlunds/atom-ruby-debugger.git
cd atom-ruby-debugger
apm link --dev . # exclude --dev if you want it to load in normal mode
npm install # might not be needed
apm install
atom --dev . # start atom. once again, optionally skip --dev
```

**Comment by joshbtn** [comment]
Thanks for the help!  `apm link --dev` Worked like a charm.  Cautionary tail for anyone else who tries the same thing, don't clone the repo right into the ~/.atom/packages directory and simply run `npm install`.  Although I feel silly now, I think these steps are a good candidate for the README under a development section of some sort.

**Comment by johanlunds** [comment]
Absolutely! It can be put under a header "Development Setup" or something like it.
